libs/hostd-types/src/api.ts

```diff
@@ -231,27 +231,26 @@ type Revenue = {
   registryWrite: string
 }
 
-// Data is a collection of metrics related to data usage.
-type Data = {
+type DataRHPMetrics = {
   // Ingress returns the number of bytes received by the host.
   ingress: number
   // Egress returns the number of bytes sent by the host.
   egress: number
 }
 
-// Contracts is a collection of metrics related to contracts.
-type Contracts = {
-  pending: number
+type ContractMetrics = {
   active: number
   rejected: number
   failed: number
+  renewed: number
+  finalized: number
   successful: number
+  pending: number // TODO: remove?
```

Comment:
@n8maninger pending is not shown here: https://github.com/SiaFoundation/hostd/issues/450, is this state gone?

Reply:
Yeah, pending was removed from the metrics. It’s usually too short to matter